tests/e2e/account/pw_register_test.py

```diff
@@ -0,0 +1,195 @@
+"""Test account registration."""
+
+import re
+
+import pytest
+from playwright.sync_api import Page, expect
+from pytest_bdd import given, parsers, scenarios, then, when
+
+scenarios('account/register.feature')
+pytestmark = pytest.mark.usefixtures('live_server')
+
+TWOFLOWER_EMAIL = 'twoflower@example.org'
+TWOFLOWER_PHONE = '+12015550123'
+TWOFLOWER_PASSWORD = 'te@pwd3289'  # nosec
+
+
+def check_recaptcha_loaded(page: Page) -> None:
+    page.wait_for_selector(
+        "#form-passwordlogin > div.g-recaptcha > div > div.grecaptcha-logo > iframe",
+        timeout=10000,
+    )
+
+
+@given("Anonymous visitor is on the home page")
+def given_anonuser_home_page(live_server, page) -> None:
+    page.goto(live_server.url)
+    expect(page).to_have_title(re.compile("Test Hasgeek"))
```

Comment:
There's no point calling re.compile at runtime. If this function accepts a plain string, you can pass one, as this is a plain text pattern.